config/pmd/ruleset.xml

```diff
@@ -62,6 +62,13 @@
 
   </rule>
 
+  <rule ref="category/java/codestyle.xml/UseExplicitTypes">
+    <properties>
+      <property name="allowLiterals" value="true" />
+      <property name="allowCtors" value="true" />
+    </properties>
+  </rule>
+
```

Comment:
what is your reason for not allowing var everywhere?

Reply:
Probably because I got pmd error while using var in code